tests/rptest/tests/datalake/schema_evolution_test.py

```diff
@@ -227,3 +385,472 @@ def run_verifier(schema: str, mapping: str):
                     root.verifier_string = 23.0
                     root.ordinal = counter()
                     """)
+
+
+class Schema(NamedTuple):
+    schema: str
+    example: Callable[[float], dict]
+    spark_rep: list[tuple]
+    trino_rep: list[tuple]
+    select: list[str]
```

Comment:
nit: mind adding comments about what each are?

nit: also maybe name it something to the effect SchemaTestCase or SchemaExpectations or something, given it's not just a schema?

Reply:
done